src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/Fabrics/StartAzureRmRecoveryServicesAsrSwitchProcessServer.cs

```diff
@@ -0,0 +1,150 @@
+﻿// ----------------------------------------------------------------------------------
+// 
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Management.RecoveryServices.SiteRecovery.Models;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery
+{
+    /// <summary>
+    ///     Initiates switching process server for load balancing.
+    /// </summary>
+    [Cmdlet(
+        VerbsLifecycle.Start,
+        "AzureRmRecoveryServicesAsrSwitchProcessServer",
+        DefaultParameterSetName = ASRParameterSets.Default,
+        SupportsShouldProcess = true)]
+    [Alias("Start-ASRSwitchProcessServer")]
```

Comment:
There is no provision of stop switching  server as it will lead configuration is ambiguous state migrated / nor migrated. That is the reason service is not supporting STOP feature so the portal and Powershell